Makefile

```diff
@@ -56,6 +56,14 @@ ALL = \
 	$(ISO_ROOT)/salt/metalk8s/runc/init.sls \
 	$(ISO_ROOT)/salt/metalk8s/runc/installed.sls \
 	\
+	$(ISO_ROOT)/salt/metalk8s/saltmaster/files/master_99-metalk8s.conf.j2 \
+	$(ISO_ROOT)/salt/metalk8s/saltmaster/files/minion_99-metalk8s.conf.j2 \
+	$(ISO_ROOT)/salt/metalk8s/saltmaster/files/salt-master-pod.yaml.j2 \
+	$(ISO_ROOT)/salt/metalk8s/saltmaster/init.sls \
+	$(ISO_ROOT)/salt/metalk8s/saltmaster/master_configured.sls \
+	$(ISO_ROOT)/salt/metalk8s/saltmaster/minion_configured.sls \
+	$(ISO_ROOT)/salt/metalk8s/saltmaster/deployed.sls \
```

Comment:
Global comment for the state would prefer `$(ISO_ROOT)/salt/metalk8s/salt/master/*`

Reply:
I don't, actually. Rather be explicit, instead of using `*`, run tests on you machine which copy files not in Git, push without those files being checked in, and things not working. Or the other way around.

In general, the stuff in `_build/root` should never come from 'somewhere' without being an explicit target.

Reply:
Aaaaah, you mean `metalk8s/salt/master/...`, not using a wildcard. All for it then!

Reply:
Yes I mean a salt directory :)